app/controllers/coronavirus_form/offer_care_qualifications_controller.rb

```diff
@@ -22,7 +22,8 @@ def submit
         allowed_values: I18n.t("coronavirus_form.questions.#{controller_name}.care_qualifications.options").map { |_, item| item.dig(:label) },
         other: @form_responses[:offer_care_qualifications_type],
         other_field: "nursing_or_healthcare_qualification",
-      )
+      ) +
```

Comment:
Since there are now quite a few validations, I'm wondering whether the syntax we've used in other controllers (where we have an array that is flattened) may be slightly more elegant?  E.g. https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/0bc813c0afd1e0ebb3f1bd58e3d67e6fe969c644/app/controllers/coronavirus_form/contact_details_controller.rb#L31-L37

Reply:
👍